FAIMS/FAIMS3
# feat: survey form layout

## JIRA Ticket

[BSS-382](https://jira.csiro.au/browse/BSS-382)

## Description

Changed the layout of the create site record form to be in a single long view.

## Proposed Changes

- updated form component to show list with headings rather than tabs
- moved errors to the end of the form

## How to Test

This was tested using the Rapid Impact Survey.

1. Navigate to a survey
2. Add a new record
3. Verify layout and form functionality
4. (optional) Open an already created record and verify layout and functionality

## Additional Information

This is an update rather than configurable. Happy to discuss the option of being configurable but it seems currently there are a few bugs in the form code and if we were to maintain both we will need to address these in each version of the component.

## Checklist
